File: AppointmentSchedulerAPI/layers/ServiceLayer/v1/Controllers/DTO/Response/AppointmentNotificationDTO.cs
using AppointmentSchedulerAPI.layers.BusinessLogicLayer.Model.Types.Notification;

namespace AppointmentSchedulerAPI.layers.ServiceLayer.v1.Controllers.DTO.Response
{
    public class AppointmentNotificationDTO : NotificationDTO
    {
        public required AppointmentNotificationCodeType Code { get; set;}

        public required Guid AppointmentUuid { get; set;}

    }
}

File: AppointmentSchedulerWeb/src/model/dtos/appointment-notification.dto.ts
import { AppointmentNotificationCodeType } from "../../view-model/business-entities/types/appointment-notification-code.types";
import { AppointmentUuidDTO } from "./appointment-uuid.dto";
import { NotificationDTO } from "./notification.dto";

export interface AppointmentNotificationDTO extends NotificationDTO {
  Code: AppointmentNotificationCodeType,
  Appointment: AppointmentUuidDTO
}
